scout/server/blueprints/clinvar/controllers.py

```diff
@@ -107,7 +107,7 @@ def _get_snv_var_form(variant_obj, case_obj):
     var_form.variations_ids.data = var_ids.split(";")[0]
     var_form.chromosome.data = variant_obj.get("chromosome")
     var_form.start.data = variant_obj.get("position")
-    var_form.stop.data = variant_obj.get("position")
+    var_form.stop.data = variant_obj.get("end")
```

Comment:
Did you check the docs or test with their stage endpoint to see if this is correct in terms of inclusive coords (off by one or not)?

Reply:
Nope. I can try!

Reply:
Tested using the current submission for cust000 on stage.
Downloaded it as json and used it in a dry-run using preClinVar:

<img width="476" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/9fad5d0f-3c3b-4d5d-ad1a-53dac1f33382">

Reply:
And before they would have failed it? 😊

Reply:
No, the dry run doesn't check that type of things it seems (I tried now). It returns success as well

Checked on their docs and they say: 
<img width="994" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/13a4448f-1720-4937-9efa-a02505eae410">

So we should be fine, no?

Reply:
Did you check the JSON summary report from the dry-run? It should be available with that sub nr right?

Reply:
Let's try like this. We can fix if they complain. It's not that they make us test better using the test endpoint